clientd3d/d3drender.c

```diff
@@ -9280,6 +9292,16 @@ Bool D3DMaterialWorldStaticChunk(d3d_render_chunk_new *pChunk)
 		SetZBias(gpD3DDevice, ZBIAS_WORLD);
 	}
 
+	if (pChunk->flags & D3DRENDER_NO_VTILE)

```

Comment:
In the spirit of repeating things less, I might write this instead as:

```
auto state = (pChunk->flags & D3DRENDER_NO_VTILE) ? D3DTADDRESS_CLAMP : D3DTADDRESS_WRAP;
IDirect3DDevice9_SetSamplerState(gpD3DDevice, 0, D3DSAMP_ADDRESSV, state);

```

Same thing for the other similar changes in this file.

Reply:
I'm not sure I love the way it turned out, though I did check in your suggestion. Setting it directly seemed more concise to me.